gdc_client/download/parser.py

```diff
@@ -75,7 +82,48 @@ def download(parser, args):
         ids.add(i['id'])
 
     client = get_client(args, args.token_file)
-    client.download_files(ids)
+    downloaded_files, errors = client.download_files(ids)
+
+    if args.retry_amount > 0:
+        files_not_downloaded = []
+
+        for uuid, _ in errors.items():
+            not_downloaded_uuid = retry_download(client, uuid, args.retry_amount, args.no_auto_retry)
+            if not_downloaded_uuid:
+                files_not_downloaded.append(not_downloaded_uuid)
+    if files_not_downloaded:
+        log.warning('Files not able to be downloaded: {}'.format(files_not_downloaded))
+
+
+def retry_download(client, uuid, retry_amount, no_auto_retry):
+
+    log.info('Retrying download {}'.format(uuid))
+
+    e = True
+    while 0 < retry_amount and e:
+        if no_auto_retry:
+            should_retry = raw_input('Retry download for {}? (y/N): '.format(uuid))
```

Comment:
oh ok, I see that you can still have it run automatically without retrying with `--retry-amount 0`.